DirectProgramming/C++SYCL_FPGA/Tutorials/Features/experimental/mmhost/src/part_3_ddr_hosts.cpp

```diff
@@ -0,0 +1,114 @@
+#include <sycl/ext/intel/fpga_extensions.hpp>
+#include <sycl/sycl.hpp>
+#include <sycl/ext/oneapi/annotated_arg/annotated_ptr.hpp>
+
+#include "exception_handler.hpp"
+
+using namespace sycl;
+using namespace ext::oneapi::experimental;
+
+using usm_buffer_location =
+    ext::intel::experimental::property::usm::buffer_location;
+
+constexpr int kBL1 = 1;
+constexpr int kBL2 = 2;
+
+struct DDR_IP{
+
+  using params = decltype(properties{
+          buffer_location<kBL1>,
+          maxburst<8>,
+          dwidth<256>,
+          alignment<32>
+          });
+  annotated_ptr<int, params> x; 
+  annotated_ptr<int, params> y;
```

Comment:
Same comment here as the README. 

```suggestion
  annotated_ptr<int, decltype(properties{
          buffer_location<kBL1>,
          maxburst<8>,
          dwidth<256>,
          alignment<32>
          })> x, y
```